### PR TITLE
Tidy - misc cleanups

### PR DESCRIPTION
**Pull request by ahgittin**

### CODE REVIEW

File: guide/misc/migrate-to-0.8.0.md [hunk 1]
```diff
@@ -29,4 +29,4 @@ $ for x in `find . -type file` ; do sed -E -i .bak -f /tmp/migrate.sed $x ; done
 $ find . -name "*.bak" -delete
 {% endhighlight %}
 
-If you encounter any issues, please [contact us](/community/).
+If you encounter any issues, please [contact us](/website/community/).
```
Comment (drigodwin) [inline guide/misc/migrate-to-0.8.0.md]: This link will be incorrect on the published website

File: LICENSE [hunk 2]
```diff
@@ -387,6 +387,12 @@ This project includes the software: Swagger UI
   Used under the following license: Apache License, version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
   Copyright (c) SmartBear Software (2011-2015)
 
+This project includes the software: Tango Project Icons
+  Available at: https://commons.wikimedia.org/wiki/File:Network-server.svg
+  Inclusive of: server.svg
+  Used under the following license: Public Domain (https://en.wikipedia.org/wiki/en:public_domain)
+  Released into the public domain by the people from the Tango! project (The Tango! Desktop Project), via Wikimedia Commons
+
```
Comment (drigodwin) [inline LICENSE]: I think [Richards PR](https://github.com/apache/brooklyn-docs/pull/167) added this.

File: guide/misc/migrate-to-0.8.0.md (hunk 1 above)
Comment (tbouron) [inline guide/misc/migrate-to-0.8.0.md]: +1. The published website is hosted at the root. `/website` is only for dev environment`

File: _includes/footer.html [hunk 3]
```diff
@@ -1,10 +1,10 @@
 <div id="footer">
     <div class="container">
         <div class="row">
-            <div class="col-md-10 text-muted">
-                Apache Brooklyn is distributed under the <a href="http://www.apache.org/licenses/LICENSE-2.0">Apache License v2.0</a>.
+            <div class="col-md-10 text-muted footer-left footer-text-next-to-buttons">
+                <p>Apache Brooklyn is distributed under the <a href="http://www.apache.org/licenses/LICENSE-2.0">Apache License v2.0</a>.</p>
             </div>
-            <div class="col-md-2">
+            <div class="col-md-2 footer-right">
```
Comment (tbouron) [inline _includes/footer.html]: I don't think those changes are needed since #173

File: _layouts/website-landing.html [hunk 4]
```diff
@@ -26,10 +26,10 @@
 <div id="footer">
     <div class="container">
         <div class="row">
-            <div class="col-md-9 text-muted">
-                Apache Brooklyn is distributed under the <a href="http://www.apache.org/licenses/LICENSE-2.0">Apache License v2.0</a>.
+            <div class="col-md-9 text-muted footer-left footer-text-next-to-buttons">
+                <p>Apache Brooklyn is distributed under the <a href="http://www.apache.org/licenses/LICENSE-2.0">Apache License v2.0</a>.</p>
             </div>
-            <div class="col-md-3">
+            <div class="col-md-3 footer-right">
```
Comment (tbouron) [inline _layouts/website-landing.html]: I don't think those changes are needed since #173

File: style/css/_footer.scss [hunk 5]
```diff
@@ -22,6 +22,10 @@ div#footer {
     background-color: $white_f5;
     text-align: center;
     font-size: 80%;
+
+    .footer-left { text-align: left; padding-left: 40px /* match list indentation etc */; }
+    .footer-text-next-to-buttons { padding-top: 4px /* used to nudge down slightly to align with button */; }
+    .footer-right { text-align: right; padding-right: 40px; }
```
Comment (tbouron) [inline style/css/_footer.scss]: I don't think those changes are needed since #173

**Comment by ahgittin** [comment]
fixed website link to use var, and kept the readme dev notes; other things that were here (footer, license) are no longer applicable

**Pull request merged**